.travis.yml

```diff
@@ -9,4 +9,6 @@ except:
 - "/^v\\d+\\.\\d+\\.\\d+$/"
 env:
   global:
-    secure: OyRVQ5TQRZQaNxe2Es516WX27G5xN1ojs6Y4X6mve5CAIScvGHoEUn9OkRJQqaaq5vDq87/RF+b6QNLttyNs34BwzsgkK1uYHtG80i975nTRNbNiWzTxZnlicle4AB0MeawtD0FP6KJzu+W78LHTsi0qbqDj2WErl2BSvvjmvTJf7d68tQBULkhrv7BWKvs6Iojr3bHdlPh9mltZXCOOyXy5aT00o8yYbBh3SwabcLfomh6Anx+6F9rYszia+PjKFHj7QUyWhaeoywJzJCBruuL8onopAbFOepcovAu5hhe93JUzmsGpaWQn77oBg4ToBUB8p9nIGKRRb5AMAPUqbtbsOLWT/VZvxhz5acmvWBhuNCbqZKU4kebHp+LEiRFQr3I/koZBh3qWF9TTwqDqPmJEGU+PbNTdGNFF7bn6AwjgZBOqchgUjdsehgBS7twfJhvGZP1yViFpp+bx48TPnSgvBHMiHUBWJdykXxlbWpTHyCAOlQ45udhFjUAR6sUOxu63QXFFd5RpSx5q+ng/Bw009DoJTfvNmXdKBz1xbkyYN7xWB9tCKV7m+1BuK8v2CEY+61VTpOHHyGeYV9etUeiK6S0WG2houZMwE5cPuC2fzqrbZ/JdlB91bBN7JWvYotf4rS/VkBXpquEUicu/l+gXu8fhjYgTShFGi93hNCI=
+  - secure: OyRVQ5TQRZQaNxe2Es516WX27G5xN1ojs6Y4X6mve5CAIScvGHoEUn9OkRJQqaaq5vDq87/RF+b6QNLttyNs34BwzsgkK1uYHtG80i975nTRNbNiWzTxZnlicle4AB0MeawtD0FP6KJzu+W78LHTsi0qbqDj2WErl2BSvvjmvTJf7d68tQBULkhrv7BWKvs6Iojr3bHdlPh9mltZXCOOyXy5aT00o8yYbBh3SwabcLfomh6Anx+6F9rYszia+PjKFHj7QUyWhaeoywJzJCBruuL8onopAbFOepcovAu5hhe93JUzmsGpaWQn77oBg4ToBUB8p9nIGKRRb5AMAPUqbtbsOLWT/VZvxhz5acmvWBhuNCbqZKU4kebHp+LEiRFQr3I/koZBh3qWF9TTwqDqPmJEGU+PbNTdGNFF7bn6AwjgZBOqchgUjdsehgBS7twfJhvGZP1yViFpp+bx48TPnSgvBHMiHUBWJdykXxlbWpTHyCAOlQ45udhFjUAR6sUOxu63QXFFd5RpSx5q+ng/Bw009DoJTfvNmXdKBz1xbkyYN7xWB9tCKV7m+1BuK8v2CEY+61VTpOHHyGeYV9etUeiK6S0WG2houZMwE5cPuC2fzqrbZ/JdlB91bBN7JWvYotf4rS/VkBXpquEUicu/l+gXu8fhjYgTShFGi93hNCI=
+  - secure: al7/mMnCmdworX2CbLwD8kSw9o6wGrNQwjFmlK00bNpKTFczzIZQJN5lzeR7airEXCq4oCCZno8eRNrhwieYKDMzw8Wq2la5J/LZSCCuzNiu8D5PYjqpUeEGyODLwdtxQs9F0JkwIZDaranKst5gKVzuFKGnYNI7XYzr8tTZvmInKS+g/oZ4nNlCIjm11jc0a+ejB9OKmDuybysLZFnlQdFaiq30fNVokimK2nSCGj8WdFXsWGk8dhwfQJsNlL7YDG9WBl6V46mhEoBqsWy3hVxBzrklpujBTgznJRQLWCeJSycbZvSZsv1rnrsL9xXZ4o+qPurAko2XRsYaoNY5Y6+5lzfoyTFJepvvXkCH3diyQfroShhpyw2riWXVTbnKL6iNkqH75w0JchBJ26oGxtmkki56OV+k2gCnIjIf4RhwYgzKW87SWerYa5vuaRMj7NI/Xphrd/ByHRbvTqK7NBxypKFkUK+rNhHYXIz3F5/5jh1lQ3kRxDCPFkaogk6VeJHRccS0dKBMlrxkiaUMwDucKmjCWRAsODk5LnmVCBXWd4Tlyxq6OCfmEI3R7alaO+RuKfG3EeG/LcdMqth/IltlHGoMAQiTGc9O4GETq/Ydel+vLYIRqFmFBFXShsd5cXpNPuTdVPVxsYSl0YLDd5UchoOXW5h+B86L+7z/V34=
```

Comment:
Why do you set 3 variables if you only use 2: client id and cliend secret

Reply:
I was using a GitHub access token, I'm going to delete it.